pkg/util/crypt_linux.go

```diff
@@ -0,0 +1,43 @@
+// +build linux
```

Comment:
I added a conditional skip to crypt_test.go ... I'll do the same for htpasswd_test.go